docs/architecture/translations.rst

```diff
@@ -68,9 +68,9 @@ We've also set a ``translated`` property being an instance of ``TranslationProxy
        class Meta:
            unique_together = ('product', 'language_code')
 
-.. note:: Don't forget to set ``unique_together`` on the ``product`` and ``language_code``, as we want to ensure there's only one translation per product per language.
+.. note:: Don't forget to set ``unique_together`` on the ``product`` and ``language_code``, there should be only one translation per product per language.
 
-.. warning:: ModelTranslation fields should take same arguments as the original fields, eg. inconsistency in ``max_length`` attribute could lead to UI bugs with translation turned on.
+.. warning:: ModelTranslation fields should take same arguments as the original, eg. inconsistency in ``max_length`` attribute could lead to UI bugs with translation turned on.
```

Comment:
It may be more comprehensible like so:
fields ~~should~~ **must always** take **the** same arguments as the ~~original~~ **existing translatable models**